app/javascript/packages/components/accordion.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRef } from 'react';
+import type { ReactNode } from 'react';
+import { useInstanceId } from '@18f/identity-react-hooks';
+
+interface AccordionProps {
+  header: string;
+
+  children: ReactNode;
+}
+
+function Accordion({ header, children }: AccordionProps) {
+  const uniqueId = useInstanceId();
+  const ref = useRef(null as HTMLDivElement | null);
+
+  return (
+    <div ref={ref}>
+      <div className="usa-accordion">
```

Comment:
I'm fine to address separately, but I noticed that we have two different types of accordion, a "bordered" version and a borderless version. The borderless version is the "default", but is not the one we [typically use](https://design.login.gov/components/accordions/). Short-term, we should just add the bordered modifier class, so that it shows with border.

```suggestion
      <div className="usa-accordion usa-accordion--bordered">
```

Before|After
---|---
![image](https://user-images.githubusercontent.com/1779930/168669172-bfee0ebb-c079-479c-ae7c-1000b80372e2.png)|![image](https://user-images.githubusercontent.com/1779930/168669281-57e78a15-0b36-4920-9885-dce4e8bffc1b.png)

Long-term, this should probably be a prop to the component, e.g.

```ts
function Accordion({ isBordered = true }) {
  const classes = ['usa-accordion', isBordered && 'usa-accordion--bordered'].filter(Boolean).join(' ');
  return <div className={classes} />;
}
```